templates/vanilla-sequoia.pkr.hcl

```diff
@@ -99,9 +104,7 @@ build {
       // Disable screensaver for admin user
       "defaults -currentHost write com.apple.screensaver idleTime 0",
       // Prevent the VM from sleeping
-      "sudo systemsetup -setdisplaysleep Off 2>/dev/null",
       "sudo systemsetup -setsleep Off 2>/dev/null",
-      "sudo systemsetup -setcomputersleep Off 2>/dev/null",
```

Comment:
The `-setsleep` flag here sets the display, computer, and disk sleep setting to the same value, so setting the display and computer sleep value here is unnecessary.